apps/galaxy/src/pages/staff/verify-email.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from "react";
+import { isEmpty } from "lodash-es";
+import EmailVerification from "@src/modules/auth/EmailVerification";
+import { useVerifyEmail, VerifyEmailRequestData } from "@src/services/verify-email/useVerifyEmail";
+import { useRouter } from "next/router";
+import Head from "@src/utils/Head";
+
+export default function EmailVerificationPage() {
```

Comment:
Why this folder named staff? Better naming should be considered.

Reply:
This is simply because the verification link sent from the backend will take the user to `/staff/verify-email` route, hence the folder.